framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bootstrap/TagHelpers/Form/AbpInputTagHelperService.cs

```diff
@@ -227,10 +227,11 @@ protected virtual void AddPlaceholderAttribute(TagHelperOutput inputTagHelperOut
 
         var attribute = TagHelper.AspFor.ModelExplorer.GetAttribute<Placeholder>();
 
-        if (attribute != null)
+        if (attribute != null || TagHelper.FloatingLabel)
         {
-            var placeholderLocalized = _tagHelperLocalizer.GetLocalizedText(attribute.Value, TagHelper.AspFor.ModelExplorer);
-
+            var placeholderLocalized = _tagHelperLocalizer.GetLocalizedText(attribute?.Value ?? string.Empty, TagHelper.AspFor.ModelExplorer);
+            if (string.IsNullOrWhiteSpace(placeholderLocalized))
+                placeholderLocalized = TagHelper.AspFor.Name;
             inputTagHelperOutput.Attributes.Add("placeholder", placeholderLocalized);
```

Comment:
Why did we add the placeholder attribute here? When I test it, it does not appear on the UI side.

```cs
<abp-input floating-label="true" label="Test" asp-for="MyProperty" />
```

![image](https://github.com/abpframework/abp/assets/58659931/471ffce8-0f81-4c08-8a4d-f5c369ba52d9)